catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/MetacardComparatorTest.java

```diff
@@ -0,0 +1,201 @@
+                .toString();
+        locWkt = "POLYGON ((30 10, 10 20, 20 40, 40 40, 30 10))";
+        nsUri = new URI("http://" + MetacardComparatorTest.class.getName());
+        resourceUri = new URI(nsUri.toString() + "/resource1.html");
+        cachedMetacard = new MetacardImpl(BasicTypes.BASIC_METACARD);
+        cachedMetacard.setContentTypeName("testContentType");
+        cachedMetacard.setContentTypeVersion("testContentTypeVersion");
+        cachedMetacard.setAttribute(Metacard.CHECKSUM, "1");
+        cachedMetacard.setCreatedDate(createdDate);
+        cachedMetacard.setEffectiveDate(effectiveDate);
+        cachedMetacard.setExpirationDate(expireDate);
+        cachedMetacard.setModifiedDate(modDate);
+        cachedMetacard.setId(cachedMetacardId);
+        cachedMetacard.setLocation(locWkt);
+        cachedMetacard.setMetadata("testMetadata");
+        cachedMetacard.setResourceURI(resourceUri);
+        cachedMetacard.setSourceId("testSourceId");
+        cachedMetacard.setTargetNamespace(nsUri);
+        cachedMetacard.setTitle("testTitle");
+        cachedMetacard.setThumbnail(cachedMetacard.getId()
+                .getBytes());
+        cachedMetacard.setDescription("testDescription");
+        cachedMetacard.setPointOfContact("pointOfContact");
+        cachedMetacard.setResourceSize("1");
+        cachedMetacard.setAttribute(Metacard.CHECKSUM, "1");
+
+        latestMetacardId = cachedMetacardId;
+        locWkt = "POLYGON ((30 10, 10 20, 20 40, 40 40, 30 10))";
+        nsUri = new URI("http://" + MetacardComparatorTest.class.getName());
+        resourceUri = new URI(nsUri.toString() + "/resource1.html");
+        latestMetacard = new MetacardImpl(BasicTypes.BASIC_METACARD);
+        latestMetacard.setContentTypeName("testContentType");
+        latestMetacard.setContentTypeVersion("testContentTypeVersion");
+        latestMetacard.setAttribute(Metacard.CHECKSUM, "1");
+        latestMetacard.setCreatedDate(createdDate);
+        latestMetacard.setEffectiveDate(effectiveDate);
+        latestMetacard.setExpirationDate(expireDate);
+        latestMetacard.setModifiedDate(modDate);
+        latestMetacard.setId(latestMetacardId);
+        latestMetacard.setLocation(locWkt);
+        latestMetacard.setMetadata("testMetadata");
+        latestMetacard.setResourceURI(resourceUri);
+        latestMetacard.setSourceId("testSourceId");
+        latestMetacard.setTargetNamespace(nsUri);
+        latestMetacard.setTitle("testTitle");
+        latestMetacard.setThumbnail(latestMetacard.getId()
+                .getBytes());
+        latestMetacard.setDescription("testDescription");
+        latestMetacard.setPointOfContact("pointOfContact");
+        latestMetacard.setResourceSize("1");
+        latestMetacard.setAttribute(Metacard.CHECKSUM, "1");
+
+    }
+
+    @Test
+    public void isSameMetacard() throws Exception {
+
+        Assert.assertEquals(true, MetacardComparator.isSame(cachedMetacard, latestMetacard));
```

Comment:
We should do an `import static` of `MetacardComparator` and use either `assertThat(isSame(cachedMetacard, latestMetacard), is(true))` or `assertTrue(isSame(cachedMetacard, latestMetacard))`.